tests/cli/test_detect_format.py

```diff
@@ -48,7 +48,7 @@ def test_ambiguous(self):
             output_file = io.StringIO()
 
             with contextlib.redirect_stdout(output_file):
-                run(self, 'detect-format', test_dir)
+                run(self, 'detect-format', '--depth', '0', test_dir)
```

Comment:
This test doesn't seem valid now, because it should be ambiguous either way. Could you check it?